assets/esm_shims.js

```diff
@@ -1,6 +1,6 @@
 // Shim globals in esm bundle
-import { fileURLToPath } from 'url'
-import path from 'path'
+import * as path from 'node:path'
```

Comment:
seems like import all is not necessary?

```suggestion
import path from 'node:path'
```

Reply:
I think it's the same, but using a default export saves `* as` and reduces bundle size.

Reply:
Sure, I'll revert it.